nuxt-pinia-tailwind/components/TabNavigation/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface TabItem {
```

Comment:
nit: could we rename this to `ITabItem` instead? Just to keep following the naming convention of interfaces in the project